packages/cashier/src/components/email-verification-empty-state/email-verification-empty-state.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import EmptyState from 'Components/empty-state';
+import { useVerifyEmail } from '../../hooks';
```

Comment:
@hamid-deriv I would suggest not, We are in a Monorepo and we shouldn’t be using import aliases as it causes problems. If you check each package you can see we have lots of unnecessary configurations just to make it work, TS, Jest, Webpack, etc.
If you have a folder in your package and you want to have import aliases for it, then that folder is a package itself. For example this `hooks` folder should be a separate package and we will move it to a separate package later on. I can add the alias for it but it will be double work to remove it again in the future, Please let me know if I should make the changes 🙏🏻🙇🏻

Reply:
We have some other aliases in cashier right now, like `Components`, `Containers`, and so on. What is the difference between the `hooks` folder and other folders we have aliases for them?
As you said, this folder will be moved to a separate package, but this is not planned and I think we can stay on the code styleguide we have in the package.

Reply:
@hamid-deriv Yes for existing ones the plan was to remove them and remove all aliases and let TS handle that, Not sure if it's still the case 😅 I will add the alias for now 🙇🏻 

---

@ali-hosseini-fs Have we decided on removing aliases yet or since we patched the `tsconfig` we gonna ignore it? 🤔

Reply:
@farzin-fs since the changes are a lot, i think for now its better to don't do that.